onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -27,14 +27,28 @@
 
 class QDQQuantizer(ONNXQuantizer):
     def __init__(self, model, per_channel, reduce_range, mode, static, weight_qType, input_qType, tensors_range,
-                 nodes_to_quantize, nodes_to_exclude, op_types_to_quantize, extra_options={}):
+                 nodes_to_quantize, nodes_to_exclude, op_types_to_quantize, op_types_to_exclude_output_quantization=[], extra_options={}):
```

Comment:
>op_types_to_exclude_output_quantization [](http://example.com/codeflow?start=76&length=39)

Please make it as an option of the exra_options

Reply:
Done